### DISCUSSION TITLE
Hardcard support

### DISCUSSION THREAD

**Discussion by Epictyphlosion** [86Box/86Box#4176] [Ideas]
In the mid-80's, Plus Development made an ISA expansion card which contained a 20MB hard drive, drive controller and adapter all in one. This meant you could add extra space in your machine without having to replace your current drive or deal with adding an HDD controller. By the end of the decade, there were dozens of clones all using different sizes of drives and promising different features.

The benefits from 86box supporting this would be plentiful, the big one being the ability to add more space without having to reconfigure other parts of your machine. One way I could see this being implemented is to add an ISA option in the drop down menu where you can select wether the drive is MFM/RLL, ATA, SCSI, etc. Selecting it would make the drive act as if it were a hardcard.

**Comment by jriwanek** [comment]
You can already add more space "without having to reconfigure other parts of your machine", and you can already add various HDC's to various machines, including MFM, ATA, SCSI and so on; all a hardcard is is a controller, with a drive strapped to it, there's literally no other difference, maybe you can explain better what you mean?
